package.json

```diff
@@ -9,15 +9,17 @@
     "prepublish": "npm run build",
     "test": "mocha"
   },
+  "engines": {
+    "node": ">=4.5.0"
```

Comment:
shouldn't this be a current LTS node 10? Why target a version which is unsupported for many months/years now?

Reply:
Actually this is not target version. It indicates minimal required to install/run package.
In general I agree that this version could be 8.x (because 6.x EOL today) but idk any reasons to do it, there are not any incompatibilities between..

Reply:
only reason I can think of is that developers who contribute to the project might be swayed away from including deps which have the engine version higher than the one in package.json for the fear of incompatibility. It is a general rule to only install packages which have their own minimal version lower than the one your project has.